core/src/sql/statements/set.rs

```diff
@@ -57,6 +57,11 @@ impl SetStatement {
 
 impl fmt::Display for SetStatement {
 	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-		write!(f, "LET ${} = {}", self.name, self.what)
+		write!(f, "LET ${}", self.name)?;
+		if let Some(ref kind) = self.kind {
+			write!(f, ": {}", kind)?;
+		}
+		write!(f, " = {}", self.what)?;
+		Ok(())
```

Comment:
Shall we add a non-regression test?

Something like https://github.com/surrealdb/surrealdb/blob/6e9d04d4d7801c359cf3b5ca4398d07efd99bb6a/core/src/sql/statements/ifelse.rs#L153